deploy/sample/build.sample.properties

```diff
@@ -31,7 +31,7 @@ modules-prefix=/
 #
 # the location of your marklogic configuration file
 #
-# config.file=${basedir}/deploy/ml-config.xml
+config.file=${basedir}/deploy/ml-config/
```

Comment:
I think I'm going to undo this. I like the idea better to start with ml-config.xml, and have people split it themselves for now..